hostscripts/clouddata/syncSLErepos

```diff
@@ -99,7 +99,7 @@ rsync_and_unpack_iso()
 for version in 8 7 6; do
```

Comment:
It doesn't look like `version` could ever be `9`.

Reply:
@nicolasbock does it matter? we don't have 9 yet available, but when we have it available we want it to be synced from ibs, until 9 is released. 

so the code is there but its the same like commenting it out. I

Reply:
done